README.md

```diff
@@ -27,7 +27,7 @@ Font Awesome, and hosted with GitHub pages.
 ## Development
 
 ```
-bundle exec jekyll serve
+bundle exec jekyll serve -w
```

Comment:
This happens by default; you should see something like `Auto-regeneration: enabled for '/Users/pmdarrow/Code/feedhenry/feedhenry.org'` when you run `bundle exec jekyll serve`.